stock_orderpoint_move_link/models/stock_warehouse_orderpoint.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 ForgeFlow S.L.
+#   (http://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
+
+class StockWarehouseOrderpoint(models.Model):
+    _inherit = "stock.warehouse.orderpoint"
+
+    def action_view_stock_picking(self):
+        result = self.env["ir.actions.act_window"]._for_xml_id(
+            "stock.action_picking_tree_all"
+        )
+        result["context"] = {}
+        picking_ids = (
```

Comment:
To avoid this search, for performances reason, simply add this domain: `[("move_ids.orderpoint_ids", "in", self.ids)]`